sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -79,7 +79,31 @@
       <version>${bigtable.version}</version>
     </dependency>
 
-    <!-- test -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.14</version>
```

Comment:
version numbers should be inherited from root-level pom, I think. See grpc above, for instance.